library/grpc-node/v1.11.3/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# syntax=docker/dockerfile:1.4
+FROM node:16.17.1-bullseye AS build
+
+ARG TARGETARCH
+
+WORKDIR /build
+COPY --link package* .
+RUN /bin/bash <<'EOF'
+set -e
+# https://github.com/grpc/grpc-node/issues/1405#issuecomment-1195903603 - linux/arm64 binaries not available
```

Comment:
They've improved this in v1.11.3 to provide Darwin arm64 prebuilt binaries but we still need to build a Linux version.